core/node/da_clients/src/eigen/verifier.rs

```diff
@@ -362,38 +365,16 @@ impl Verifier {
     }
 
     fn decode_bytes(&self, encoded: Vec<u8>) -> Result<Vec<u8>, String> {
-        // Ensure the input has at least 64 bytes (offset + length)
-        if encoded.len() < 64 {
-            return Err("Encoded data is too short".to_string());
+        let output_type = [ParamType::Bytes];
+        let tokens: Vec<Token> = ethabi::decode(&output_type, &encoded)
+            .map_err(|_| "Incorrect result on contract call")?;
+        let token = tokens
+            .get(0)
+            .ok_or_else(|| "Incorrect result on contract call")?;
```

Comment:
Another clippy suggestion:
```suggestion
            .ok_or("Incorrect result on contract call")?;
```